phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/common/control/PhoenicisView.java

```diff
@@ -0,0 +1,49 @@
+package org.phoenicis.javafx.components.common.control;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import org.phoenicis.javafx.components.common.skin.PhoenicisViewSkin;
+
+/**
+ * A base component for a view of the Phoenicis application
+ *
+ * @param <C> The control type of the component
+ * @param <S> The skin type of the component
+ */
+public abstract class PhoenicisView<C extends PhoenicisView<C, S>, S extends PhoenicisViewSkin<C, S>>
+        extends ControlBase<C, S> {
+    /**
+     * A property containing the initialization state of this view
+     */
+    private final BooleanProperty initialized;
```

Comment:
Shouldn't this be `isInitialized`?

Reply:
I don't think so. The corresponding getter method is named `isInitialized()`.

Reply:
Fine.